crates/node/src/interactive_cli/admin_dashboard.rs

```diff
@@ -0,0 +1,18 @@
+use clap::Args;
+use eyre::Result;
+use webbrowser;
+
+#[derive(Debug, Args)]
+pub struct AdminDashboardCommand {
+    #[clap(required = true)]
+    port: u16,
+}
+
+impl AdminDashboardCommand {
+    pub fn run(&self) -> Result<()> {
+        let url = format!("http://localhost:{}/admin-dashboard", self.port);
```

Comment:
take in the argument `&Node` here, on which you can add the server config, because we [knew](https://github.com/calimero-network/core/blob/50dbf0736a55ed0c48328ef524d3e472707b8677/crates/node/src/lib.rs#L70) it when [constructing](https://github.com/calimero-network/core/blob/50dbf0736a55ed0c48328ef524d3e472707b8677/crates/node/src/lib.rs#L160) `Node`